### DISCUSSION TITLE
How can I use '--return_predictions' options in CLI

### DISCUSSION THREAD

**Discussion by jeidi98** [open-edge-platform/anomalib#2168] [Q&A] [answered]
Hello,

I use anomalib 1.2.0 version.
I want to use --return_predictions option.
'--return_predictions true > result.txt' and '--return_predictions true' are not work properly.
How can I use the option?
![image](https://github.com/openvinotoolkit/anomalib/assets/80377850/d76d03a0-b445-4f10-b30d-62f29b95a6f2)

Thank you.

**Comment by abc-125** [comment, marked as answer]
Hello,
I believe `return_predictions` here is a LightningCLI parameter, so you can follow [these instructions](https://github.com/Lightning-AI/pytorch-lightning/discussions/10509) to save predictions into a file.
